README.md

```diff
@@ -337,15 +337,16 @@ Multiple proxies support the gRPC-web protocol.
 	$ docker-compose up -d node-server envoy commonjs-client
 	```
 
-2. You can also try the [gRPC-web Go proxy][].
+2. [Nginx](https://www.nginx.com/) has a grpc-web module ([doc](https://nginx.org/en/docs/http/ngx_http_grpc_module.html), [announcement](https://www.nginx.com/blog/nginx-1-13-10-grpc/)). It seems to work with simple configs, according to [user feedbacks](https://github.com/grpc/grpc-web/discussions/1322)), even with server-streaming support ([user feedback](https://github.com/grpc/grpc-web/issues/1391)).
```

Comment:
Neither of these pages mention gRPC-web - are we sure they are referring to gRPC-web and not gRPC?